cfme/utils/tests/test_simple_locators.py

```diff
@@ -1,11 +1,12 @@
 import pytest
-
+from cfme.fixtures.soft_assert import base64_from_text
```

Comment:
shouldn't this be moved to some module ?

Reply:
perhaps later, first i wanted to sort out the direct issues